public/js/chatbotForm.js

```diff
@@ -0,0 +1,60 @@
+function validateForm() {
+    const name = document.getElementById('name').value;
+    const systemPrompt = document.getElementById('system_prompt').value;
+    const model = document.getElementById('model').value;
+    const adapter = document.getElementById('adapter').value;
+
+    if (name === "" || systemPrompt === "" || model === "" || adapter === "") {
+        return false;
+    }
+
+    return isValidInput(name) && isValidInput(systemPrompt);
+
+}
+
+function isValidInput(input) {
+    const sqlInjectionPattern = /['";-]/; 
+    return !sqlInjectionPattern.test(input);
+}

```

Comment:
Good catch on making the decision to implement this. But I think we should also make sure this anti sql injection is implemented on the backend, and not just via JS on the frontend. Perhaps in the service or repository `create_bot` function?

With that being said, however, if you have tested the SQL injection and it is 100% safe then we don't need to add anything else :)

Reply:
Yes, I already checked, and it's safe from SQL injection because SQLAlchemy handles that. However, if we were using raw SQL, it could be vulnerable. But for now, since we're not using raw SQL, it's safe.

Reply:
all good then, thanks for the confirmation!